airflow/cli/cli_parser.py

```diff
@@ -30,7 +30,6 @@
 from airflow.cli.commands.legacy_commands import check_legacy_command
 from airflow.configuration import conf
 from airflow.exceptions import AirflowException
-from airflow.executors.executor_constants import CELERY_EXECUTOR, CELERY_KUBERNETES_EXECUTOR
```

Comment:
Why did you stop using these constants?

Reply:
not a particularly good reason, will add back

Reply:
for context,  in [a previous change](https://github.com/apache/airflow/commit/d752d92a4d8ef0dee93519397e76c1ecb0eda548#diff-521fc27e2c47783cd582bbe615daeba4b6d10eee86f7f923b72e5adbc15f8eb1), when i thought that CeleryKubernetesExecutor should be consistently set across the cluster, i had added one of them.

in _this_ pr though i was essentially reverting that change.

n my memory the constants were not use prior to my change (but in actuality, the constant was used prior to my prev change -- in the if statement but not in the logging). 

`CeleryExecutor` will almost certainly never change.  and even if it were to, i think you can make a case that the using the  constant arguably impairs readability and _searchability_.  there are so many hardcoded references to CeleryExecutor in docs and in code -- you won't be able to get away with just changing the constant.

in any case, i'll use the constant :)